src/main/java/nl/novi/techiteasy/controllers/RemoteControllerController.java

```diff
@@ -0,0 +1,31 @@
+package nl.novi.techiteasy.controllers;
+
+import nl.novi.techiteasy.dtos.RemoteControllerDto;
+import nl.novi.techiteasy.dtos.RemoteControllerInputDto;
+import nl.novi.techiteasy.services.RemoteControllerService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+public class RemoteControllerController {
+    private final RemoteControllerService remoteControllerService;
+
+    public RemoteControllerController(RemoteControllerService remoteControllerService) {
+        this.remoteControllerService = remoteControllerService;
+    }
+
+    @GetMapping("/remotecontrollers/{id}")
+    public ResponseEntity<Object> getRemotecontrollerById(@PathVariable Long id) {
+        RemoteControllerDto remoteController = remoteControllerService.getRemoteControllerById(id);
+        return ResponseEntity.ok().body(remoteController);
+    }
+
+    @PostMapping("remotecontrollers")
+    public ResponseEntity<Object> createRemoteController(@Valid @RequestBody RemoteControllerInputDto remoteControllerInputDto) {
```

Comment:
Door hier een "BindingResult" toe te voegen aan je parameters, kun je validatie errors meteen afvangen en communiceren naar de gebruiker. Je kunt dit terug zien in het voorbeeld dar RobertJan in de les gebruikte.